app/src/marketplace/components/ProductTypeChooser/index.jsx

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import * as React from 'react'
+import cx from 'classnames'
+import { Row, Col } from 'reactstrap'
+import { Link } from 'react-router-dom'
+import { Translate, I18n } from 'react-redux-i18n'
+
+import standardProductImage from '$mp/assets/product_standard.png'
+import standardProductImage2x from '$mp/assets/product_standard@2x.png'
+import communityProductImage from '$mp/assets/product_community.png'
+import communityProductImage2x from '$mp/assets/product_community@2x.png'
+
+import links from '../../../links'
+
+import styles from './productTypeChooser.pcss'
+
+const ProductTypeChooser = () => (
+    <div className={styles.root}>
+        <div className={styles.pageTitle}>
+            <Translate value="productTypeChooser.title" />
+        </div>
+        <Row className={styles.row}>
+            <Col xs={12} md={6}>
```

Comment:
We could just start to use CSS grid / breakpoints instead of using `reactstrap` anymore